CryptoPrices/Sources/Data/Sources/NetworkExtension/CoinRoute.swift

```diff
@@ -47,8 +46,7 @@ extension CoinRoute: Route {
             finalPayload.removeValue(forKey: "id")
             return finalPayload
             // TODO: Update when implement
-        default:
-            return nil
+        default: return nil
```

Comment:
We won't have any update anytime soon, can remove `// TODO: Update when implement` I think 🙏